assignment.md

```diff
@@ -11,63 +11,113 @@ If the version control concepts or `git` commands are still confusing to you, co
-        # git command and output here
+        git log -p README.md 
+        commit 00e8ae51a08a0f3754ab228f62b51651595c4426
+        Author: Xihe Xie <axiezai@gmail.com>
+        Date:   Mon Dec 28 16:01:51 2020 -0500
+
+            readme and try new size
+
+        diff --git a/README.md b/README.md
+        new file mode 100644
+        index 0000000..dbad2e5
+        --- /dev/null
+        +++ b/README.md
+        @@ -0,0 +1 @@
+        +# Course GitHub Pages repo
         ```
     
     - c. What was the commit message associated with the last modification to the `-Week 2` line of `index.md` file? (Hint: use `git blame` and `git show`, check out their help pages if you are not sure what to do)
+      > swapped 2 weeks
 
  5. In the course website repository (or another repository you want to clone from GitHub), modify one of its existing files, do not `git add` or `git commit` yet:
      - a. What happens when you do `git stash`, be specific about what happens to your uncommitted changes after `stash`.?
-       > Answer here
+       > My changes are not creating uncommitted files; instead, they are saved or queued up elsewhere (on stack) for returning to them at a later time. My working tree is clean.
      - b. What do you see when you run `git log --all --oneline` after stashing your modification?
-       > Answer here
+       > I see the stashed changes as work in progress; that's the top line of the output.
      - c. Look up what `git stash drop` and `git stash pop` does, either Google or use `--help`. Run `git stash drop` to undo what you did with `stash`, in what scenario might `stash`-ing, `pop`-ing, and `drop`-ing be useful?
-       > Answer here
+       > Stashing is useful when I'm not yet ready to commit my changes (I'm in a draft mode and have not finished but need to work on another file in the same repo before I'm done making changes). Dropping is useful when I'm done working with a stash and don't want to commit. Popping is useful when I'm done working with a stash and do want to commit.
 
  6. One common mistake when learning `git` is to commit large files that should not be managed by `git` or adding sensitive information like security keypairs for Amazon Cloud Services. Navigate back to your assignment repository and try adding a random text file to the repository (use `touch`), making some modifications and commits to your repository, and deleting that file with `rm`. But deleting with `rm` will not delete that file's recorded git history, therefore we need to do a little more. You may want to look at [this](https://help.github.com/articles/removing-sensitive-data-from-a-repository/))
 
-    > Perform the commits and deletion, this question will be graded based on your commit history on GitHub. I should see commits for when you added your file and made modifications, and I should see a commit for when you deleted the files. 
+    > I accidentally did this on main the first time around, but we fixed it together later.
```

Comment:
I think when we did this together, we only used `git rm --cached` to make sure there was nothing wrong with ur git repo and didn't include the entire command for filtering and replacing text. 

So your "iamlisa" is still visible in your commit history files, it's my bad for taking the easier route when we did this together, but for future reference... remove using the full command to erase that text from your history.

Reply:
I see! I usually don't ever put anything sensitive on, but I'll keep that in mind! I still don't know how to look at whether it's visible though.